ps_checkout.php

```diff
@@ -1560,34 +1381,10 @@ public function hookDisplayPaymentByBinaries(array $params)
      */
     private function getCheckoutPageUrl()
     {
-        /** @var \PrestaShop\Module\PrestashopCheckout\ShopContext $shopContext */
-        $shopContext = $this->getService(\PrestaShop\Module\PrestashopCheckout\ShopContext::class);
-
-        if ($shopContext->isShop17()) {
-            return $this->context->link->getPageLink(
-                'order',
-                true,
-                (int) $this->context->language->id
-            );
-        }
-
-        // PrestaShop 1.6 legacy native one page checkout
-        if (1 === (int) Configuration::get('PS_ORDER_PROCESS_TYPE')) {
-            return $this->context->link->getPageLink(
-                'order-opc',
-                true,
-                (int) $this->context->language->id
-            );
-        }
-
-        // PrestaShop 1.6 standard checkout
         return $this->context->link->getPageLink(
```

Comment:
It's a tiny detail but `$this->context->link->getPageLink('order')` would've been enough